From 8a57f280f7297009d13816904c2544e8b2e436db Mon Sep 17 00:00:00 2001
From: Li Wang <li.wang@windriver.com>
Date: Wed, 22 Apr 2015 17:23:31 +0800
Subject: [PATCH] kgdb Kconfig: no depend on USB module

when kgdb use usb keyboard, it must depend on usb built-in kernel.
if compile kgdb with usb module, it happens the follow error:
kernel/built-in.o: In function `kdb_get_usb_char':
(.text+xxx): undefined reference to `usb_poll_irq'

Signed-off-by: Li Wang <li.wang@windriver.com>
---
 lib/Kconfig.kgdb |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/Kconfig.kgdb b/lib/Kconfig.kgdb
index 3f64f77..33fd9ac 100644
--- a/lib/Kconfig.kgdb
+++ b/lib/Kconfig.kgdb
@@ -100,7 +100,7 @@ config KDB_CONTINUE_CATASTROPHIC
 
 config KDB_USB
 	bool "KGDB_KDB: Allow usb input device devices"
-	depends on KDB_KEYBOARD && USB
+	depends on KDB_KEYBOARD && USB=y
 	default n
 	help
 	  Activate the support for using a USB keyboard with KDB.  The
-- 
1.7.5.4

